Monika After Story/game/zz_hangman.rpy

```diff
@@ -406,7 +406,7 @@ label mas_hangman_game_preloop:
 
     # setup positions
     show monika at t21
-    if store.mas_globals.dark_mode:
+    if mas_current_background.isFltDay():
```

Comment:
dark mode is the correct check here, we only change UI elements based on the dark mode setting.

Reply:
Reverted.